components/Typography/docs/dynamic-type.md

```diff
@@ -0,0 +1,26 @@
+### Dynamic Type
+
+#### Overview
+Typography library provides APIs that allow users to support Dynamic Type feature in Material Components. For those components with Dynamic Type feature supported, text size changes based on the Dynamic Type setting provided by the user.
```

Comment:
Consider opening this document with a statement of the fact that we support Dynamic Type:

```
Material Components for iOS supports the iOS Dynamic Type feature on all [Material-supported iOS versions](../../../docs/build-env#ios).
```

Reply:
Addressed in 10ba4603d75166865c24e1e077cfa075659efb83